gnovm/cmd/gno/mod.go

```diff
@@ -208,15 +247,72 @@ func execModTidy(args []string, io commands.IO) error {
 	return nil
 }
 
-// getGnoImports returns the list of gno imports from a given path.
+func execModWhy(args []string, io commands.IO) error {
+	if len(args) < 1 {
+		return flag.ErrHelp
+	}
+
+	wd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	fname := filepath.Join(wd, "gno.mod")
+	gm, err := gnomod.ParseGnoMod(fname)
+	if err != nil {
+		return err
+	}
+	entries, err := os.ReadDir(wd)
+	if err != nil {
+		return err
+	}
+
+	importsMap := make(map[string][]string)
+	for _, e := range entries {
+		filename := e.Name()
+		if ext := filepath.Ext(filename); ext != ".gno" {
+			continue
+		}
+		if strings.HasSuffix(filename, "_filetest.gno") {
+			continue
+		}
```

Comment:
We are already ignoring `_filetests` at many places. Even if I agree to not ignore `_filetests` here, it would make this command inconsistent with other ones. IMO modifying all the instances is out of scope for this PR.

Maybe open an issue to discuss more about this? If we all agree, I will open another PR to fix all occurrences.

Reply:
Sure, sounds good.